toonz/sources/toonzlib/toonzscene.cpp

```diff
@@ -318,6 +318,14 @@ void ToonzScene::clear() {
 void ToonzScene::setProject(std::shared_ptr<TProject> project) {
   assert(project);
   m_project = project;
+  m_standAlone = true;
+  for (int i = 0; i < project->getFolderCount(); i++) {
+    if (project->getFolder(project->getFolderName(i), true)
+            .isAncestorOf(m_scenePath)) {
+      m_standAlone = false;
+      break;
+    };
+  }
```

Comment:
As written in [my previous comment](https://github.com/opentoonz/opentoonz/issues/6066#issuecomment-3226673704), users may add `scenes.xml` manually in order to make any folder to belong the project.

Instead of checking `m_standalone` here, I think the detection of standalone can be obtained [this part](https://github.com/opentoonz/opentoonz/blob/master/toonz/sources/toonzlib/tproject.cpp#L1071-L1072) in the function `TProjectManager::loadSceneProject()` .